docs/source/release-notes/workarea-3-5-0.html.md

```diff
@@ -6,7 +6,13 @@ excerpt: Release notes for Workarea 3.5.0
 Workarea 3.5.0 Release Notes
 ======================================================================
 
-__Upgrading to Workarea 3.5?__ See also [Workarea 3.5.0 Upgrade Guide](/upgrade-guides/workarea-3-5-0.html) and [Workarea Upgrade](https://plugins.workarea.com/plugins/upgrade).
+---
+
+__Prefer a video?__ &mdash; Watch the [Workarea 3.5 Community Preview](https://vimeo.com/workarea/review/370155912/3384f63954).
+
+__Upgrading to Workarea 3.5?__ &mdash; Check out the [Workarea 3.5 Upgrade Guide](/upgrade-guides/workarea-3-5-0.html), and be sure to use the [Workarea Upgrade](https://github.com/workarea-commerce/workarea-upgrade) tool.
```

Comment:
The Workarea Upgrade repo is private. Are we opening that up? If not, I'll need to remove the reference to it here.

Reply:
Going to leave private for now, since no one in the public is running v3.4 and upgrading to v3.5. I guess we should take it out, no one would have access to this repo.

Reply:
I removed the reference to Workarea Upgrade